include/circt/Dialect/FIRRTL/Passes.td

```diff
@@ -114,6 +114,19 @@ def CreateSiFiveMetadata : Pass<"firrtl-emit-metadata", "firrtl::CircuitOp"> {
   ];
 }
 
+def WireDFT : Pass<"firrtl-dft", "firrtl::CircuitOp"> {
+  let summary = "Wires test enables to clock gates for DFT infrastructure";
+  let description = [{
+    This pass will take a 1-bit signal targeted by
+    `DFTTestModeEnableAnnotation` and wires it to the `test_en` port of every
+    module named `EICG_wrapper`. This will create ports in any intermediate
```

Comment:
This description makes me sad.  (not the text, the text is fine, but the functionality)